cabal-install/tests/fixtures/init/golden/cabal/cabal-lib-and-exe-with-comments.golden

```diff
@@ -80,31 +76,36 @@ executable foo
 
     -- Other library packages from which modules are imported.
     build-depends:
-        base ^>=4.13.0.0,
-        containers ^>=5.7.0.0,
-        unordered-containers ^>=2.7.0.0
+        base,
+        y
 
     -- Directories containing source files.
-    hs-source-dirs:   app
+    hs-source-dirs:   exe
 
     -- Base language which the package is written in.
     default-language: Haskell2010
 
-test-suite foo-test
+test-suite y-test
     -- Base language which the package is written in.
     default-language: Haskell2010
 
+    -- Modules included in this executable, other than Main.
+    -- other-modules:
+
+    -- LANGUAGE extensions used by modules in this package.
+    -- other-extensions:
+
     -- The interface type and version of the test suite.
     type:             exitcode-stdio-1.0
 
     -- Directories containing source files.
-    hs-source-dirs:   tests
+    hs-source-dirs:   test
 
     -- The entrypoint to the test suite.
-    main-is:          MyLibTest.hs
+    main-is:          Main.hs
 
     -- Test dependencies.
     build-depends:
-        base ^>=4.13.0.0,
-        containers ^>=5.7.0.0,
-        unordered-containers ^>=2.7.0.0
+        base,
```

Comment:
FYI, these `base` dependencies are artificially mangled so that we can check that the golden files are generated homogeneously across GHC versions. There are other unit tests that cover the correctness of the `base` dependency version inferrance, and the user will see `base ^>=4.14.0.0` etc. in practice. 

As they are, the original golden files are actually _incorrect_ for any GHC < 8.8 in more ways than one: note the version commentary generated above. It's off by 1!

Reply:
Is this comment resolved? It's the only one I can see in https://github.com/haskell/cabal/pull/7344/files view.

Reply:
Yes this is resolved.